vignettes/00-introduction.Rmd

```diff
@@ -1,3 +1,36 @@
 # (PART\*) Section 1: Vignettes {-}
 
-# Vignettes Introduction
\ No newline at end of file
+# Vignettes Introduction
+
+
+These vignettes provide a **quick start** tutorial - they quickly show you how to access and perform ‘hello world’ type introductions to different types of data in the TERRA Reference datasets. These vignettes are designed so that they don’t require any user accounts or passwords to access to the data so that they can be run as easily as possible.
+
+The vignettes currently include:
+* trait data
+* weather data
+* image data
+* how to use combine trait, weather, and image data in a single analysis
+
+In order to make these vignettes as easy to use as possible, we only use sample of the full dataset is available without an account; specifically we have made data from thirty two plots in each of two seasons available. Although we want to make these data broadly available, we want to keep track of who is using them and collect emails so that we can contact you.
+
+
+## Prerequisites
+
+### R
+
+In order to use the vignettes, you will need to have R installed along with the extra packages that are used. 
+All of the packages except 'trait' are available on CRAN and can be installed using the `install.packages` funciton:
+
+```r
+install.packages('tidyverse', 'jsonlite', 'knitr', 'lubridate', 'raster', 'sf')
+```
+
+Although 'traits' is also on CRAN, you need to download the TERRA REF version from GitHub to handle larger datasets: 
+
+```r
+install.packages('devtools')
+devtools::install_github('terraref/traits')
+```
+### Python
+
+... need instructions on how to install terrautils! ... 
```

Comment:
@Chris-Schnaufer or @KristinaRiemer could one of you put instructions for installing terrautils here?

Reply:
There are instructions in the images vignette (https://terraref.github.io/tutorials/retrieve-source-rgb-image-files.html), do we want them in both places? I think it makes sense for this intro page to not have installation instructions, and to leave specific install stuff in the relevant vignette.

Reply:
And I'm only thinking that because it'll be easier to keep installation instructions up to date in each of the vignettes. But I don't have super strong feelings about it.

Reply:
I think having it in the introduction would be better; otherwise it will be necessary to make sure that installation instructions are in every vignette. 

 ```suggestion
pip install terrautils
```